.github/workflows/terraform-plan.yml

```diff
@@ -191,6 +196,17 @@ jobs:
       - name: Generate plaintext plan
```

Comment:
do we still need this step if we're going to be generating the json version of the plan in the next step? I assume this is still useful for logging purposes, but wanted to make sure.